tasks/package.json

```diff
@@ -15,7 +15,7 @@
     "@headlessui/react": "1.7.19",
     "@helpwave/api-services": "workspace:*",
     "@helpwave/common": "workspace:*",
-    "@helpwave/proto-ts": "0.60.0-1191f09.0",
```

Comment:
tasks should not depend on the protos. As it should be independent and only use them through the api-services. If this causes problems, we should open a separate issue.